### ISSUE TITLE
Supersede Open Tracing with mpTelemetry-1.0

### ISSUE THREAD

**Issue by tevans78** [OpenLiberty/open-liberty#23422]
We need to update the following docs to say that all the open tracing features have been stabilized and superseded by mpTelemetry-1.0

https://www.ibm.com/docs/en/was-liberty/base?topic=liberty-stabilized-features-feature-capabilities
https://www.ibm.com/docs/en/was-liberty/base?topic=liberty-superseded-features

Also need to make a small change to the feature files before GA: https://github.com/OpenLiberty/open-liberty/issues/24045

**Comment by Emily-Jiang** [comment]
@dmuelle @chirp1 We will need to update the doc for the above two pages. We need to work out which Open Liberty version first.

For the stabolization page:
Fix pack when stabilized | Category | Stabilized function | Strategic alternative
-- | -- | -- | --
xxxx |MicroProfile Tracing| MicroProfile OpenTracing: mpOpenTracing-1.0, mpOpenTracing-1.1, mpOpenTracing-1.2, mpOpenTracing-1.3, mpOpenTracing-2.0, mpOpenTracing-3.0 and OpenTracing features:  opentracing-1.0, opentracing-1.1, opentracing-1.2, opentracing-1.3, opentracing-2.0| Use MicroProfile Telemetry feature such as mpTelemetry-1.0. For more information, see the migration document from MicroProfile OpenTracing to MicroProfile Telemetry 1.0.

For the supercede page:
Category | Superseded feature | Superseding feature or feature capability | Dependent feature removed
-- | -- | -- | --
MicroProfile OpenTracing | mpOpenTracing-1.0, mpOpenTracing-1.1, mpOpenTracing-1.2, mpOpenTracing-1.3, mpOpenTracing-2.0, mpOpenTracing-3.0, opentracing-1.0, opentracing-1.1, opentracing-1.2, opentracing-1.3, opentracing-2.0 | mpTelemetry-1.0 | none

**Comment by tevans78** [comment]
@Emily-Jiang The mpTelemetry-1.0 UFO says that _all_ versions of open tracing would be stabilized and superseded...

- mpOpenTracing-1.0
- mpOpenTracing-1.1
- mpOpenTracing-1.2
- mpOpenTracing-1.3
- mpOpenTracing-2.0
- mpOpenTracing-3.0
- opentracing-1.0
- opentracing-1.1
- opentracing-1.2
- opentracing-1.3
- opentracing-2.0

**Comment by Emily-Jiang** [comment]
ah, I should list them all. I thought I just need to list the latest.

**Comment by dmuelle** [comment]
Hi, I opened https://github.ibm.com/websphere/liberty-docs/issues/3204 to track the doc updates as described here for WebSphere Liberty docs.

Does this stabilization have POC approval yet?  I've labeled the docs issue `POC Required` but if it already has or doesn't need approval I can change to `POC Approved`.

On the Open Liberty side, we will add stabilization statements to the OpenTracing and MicroProfile OpenTracing pages- **unless** there are updates planned for the metatype that will add these statements as part of the autogenerated feature descriptions. If the stabilization is the same release as MP OpenTelemetry, we can track those updates in https://github.com/OpenLiberty/docs/issues/5258. Otherwise, I'll open a new issue.
